From broonie@opensource.wolfsonmicro.com  Wed Jan 27 14:36:56 2010
From: Mark Brown <broonie@opensource.wolfsonmicro.com>
Date: Mon, 18 Jan 2010 13:18:34 +0000
Subject: USB: s3c-hsotg: Export usb_gadget_register_driver()
To: Greg Kroah-Hartman <gregkh@suse.de>, David Brownell <dbrownell@users.sourceforge.net>, Ben Dooks <ben-linux@fluff.org>
Cc: linux-usb@vger.kernel.org, Mark Brown <broonie@opensource.wolfsonmicro.com>
Message-ID: <1263820715-12483-1-git-send-email-broonie@opensource.wolfsonmicro.com>


USB gadget controller drivers normally export their driver registration
function, allowing modular builds of the individual gadget drivers so
do so for s3c-hsotg, fixing builds.

Signed-off-by: Mark Brown <broonie@opensource.wolfsonmicro.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/s3c-hsotg.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/s3c-hsotg.c
+++ b/drivers/usb/gadget/s3c-hsotg.c
@@ -2582,6 +2582,7 @@ err:
 	hsotg->gadget.dev.driver = NULL;
 	return ret;
 }
+EXPORT_SYMBOL(usb_gadget_register_driver);
 
 int usb_gadget_unregister_driver(struct usb_gadget_driver *driver)
 {
